index.rst

```diff
@@ -15,5 +10,241 @@ Change Record
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
+planning package. Each epic has a clear description, well defined deliverables, and
+relationships with a planning package (an epic *implements* a PP), and a milestone (an epic
+*blocks* a milestone). Epics are resource loaded, and have start and end dates defined.
+
+Each epic is broken into smaller chunks of work, called *stories*. Stories are primarily used
+to define and manage the short-term activities of individual developers.
+
+Epics and stories are used to track all software work, as well as work on delivering hardware and
+services that are tightly coupled with software (example: "Deliver OpenStack-based Test Cluster
+running RedHat 6.1".) Details of work on standalone activities do not have to be captured on daily
+bases through stories (example: "Deliver 10Gbit link between Chile and USA")
+
+.. _cycles-and-releases:
+
+Cycles and Releases
+~~~~~~~~~~~~~~~~~~~
+
+The work is executed through 6-month *cycles*. The scope of work for each cycle is driven by the
+Baseline Plan, in particular by milestones. Each cycle ends with a new software release. Releases are
+thoroughly tested and documented.
+
+Cycles are planned through *epics*. Each epic must be defined such that it can be completed
+within a single cycle.
+
+Once a plan is defined for a given cycle, it is loaded to Primavera and changed-controlled. A plan for
+a cycle is loaded to Primavera during the month proceeding the start of the cycle.
+
+Any non-trivial adjustments to the plan that affect scope, schedule or budget must be approved
+by CCB.
+
+Level of Effort
+---------------
+
+Two general rules are applied for the DM Team labor when determining whether an activity should be
+treated as LOE or non-LOE:
+
+* All activities that have a clear deliverable are tracked through non-LOE. Examples of non-LOE
+  activities include writing a new piece of software, purchasing new equipment, or adding new
+  machines to the cluster.
+
+* Activities that can not be predicted, as well as short, recurring routine activities are considered LOE.
+  Examples of LOE activities include replacing faulty disk drive, cluster maintenance, or writing monthly
+  status report.
+
+For planning purposes, on average 70% of available time of each software developer is assumed to be
+available for "pure, uninterrupted coding", and the remaining 30% is considered as "overhead", and tracked
+explicitly as LOE.
+
+A small fraction of all DM labor is performed by contractors. The contractor labor is typically managed
+as LOE, directly in dollars, rather than in epics and stories.
```

Comment:
But if contractors have defined deliverables (and shouldn't they always?), shouldn't those be epics?